3_visualize/src/visualize_data.R

```diff
@@ -1,16 +1,8 @@
-
-library(dplyr)
-library(readr)
-library(stringr)
-library(sbtools)
-library(whisker)
-
-
-
 # Create a plot
-plot_rmse_data = function() {
-  eval_data = read_csv("2_process/out/model_summary_results.csv")
-  png(file = '3_visualize/out/figure_1.png', width = 8, height = 10, res = 200, units = 'in')
+plot_rmse_data = function(data_file = "2_process/out/model_summary_results.csv") {
```

Comment:
And again, let's pass the output filepath as an argument

Reply:
Done!